foundry/external_data_architectures.py

```diff
@@ -25,3 +26,23 @@ def __getitem__(self, idx):
         
         return item
     
+class FoundryDataset_TF(Sequence):
```

Comment:
Could you split TF and Pytorch into separate files? That way both do not need to be installed to use one of the loaders.